capella/download_archive.py

```diff
@@ -25,8 +25,11 @@ def get_directory_structure(url: str) -> list[str]:
 
 
 if __name__ == "__main__":
+    desired_version = sys.argv[1]
+    print(f"Installing Capella {desired_version}")
+
     versions = get_directory_structure(CAPELLA_INDEX_URL)
-    if not (capella_version := os.getenv("CAPELLA_VERSION")) and versions:
+    if not (capella_version := desired_version) and versions:
         capella_archive_path = versions[-1]
     else:
```

Comment:
When you have the default value set in the `Dockerfile`, we should remove the fallback here.

Reply:
Indeed. I just "fixed" it.

Now it's "fixed and improved" :smile: